x-pack/metricbeat/metricbeat.reference.yml

```diff
@@ -265,6 +265,17 @@ metricbeat.modules:
   tenant_id: '${AZURE_TENANT_ID:""}'
   subscription_id: '${AZURE_SUBSCRIPTION_ID:""}'
 
+  - module: azure
+    metricsets:
+    - database_account
+    enabled: true
+    period: 300s
+    client_id: '${AZURE_CLIENT_ID:""}'
+    client_secret: '${AZURE_CLIENT_SECRET:""}'
+    tenant_id: '${AZURE_TENANT_ID:""}'
+    subscription_id: '${AZURE_SUBSCRIPTION_ID:""}'
```

Comment:
```suggestion
- module: azure
  metricsets:
  - database_account
  enabled: true
  period: 300s
  client_id: '${AZURE_CLIENT_ID:""}'
  client_secret: '${AZURE_CLIENT_SECRET:""}'
  tenant_id: '${AZURE_TENANT_ID:""}'
  subscription_id: '${AZURE_SUBSCRIPTION_ID:""}'
```

Reply:
@kaiyan-sheng this was fixed in the earlier commits